packages/theme-patternfly-org/components/tableOfContents/tableOfContents.css

```diff
@@ -0,0 +1,15 @@
+.ws-toc {
+  top: var(--pf-global--spacer--md);
+  order: 1;
+  position: sticky;
+  position: -webkit-sticky; /* Safari */
```

Comment:
maybe should add postcss autoprefixer to webpack instead

Reply:
Done